src/main/java/org/czekalski/petapiintegration/controllers/AuthorizationController.java

```diff
@@ -1,61 +1,160 @@
 package org.czekalski.petapiintegration.controllers;
 
 
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.czekalski.petapiintegration.apiClient.v1.dto.AnimalDto;
+import org.czekalski.petapiintegration.apiClient.v1.dto.AnimalsListDto;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.beans.factory.annotation.Value;
 import org.springframework.security.core.Authentication;
 import org.springframework.security.core.context.SecurityContextHolder;
 import org.springframework.security.oauth2.client.OAuth2RestTemplate;
-import org.springframework.stereotype.Controller;
 import org.springframework.ui.Model;
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.RestController;
 
+import java.io.IOException;
+
 @RestController
 public class AuthorizationController {
 
-
-
     @Value("${messages.base-uri}")
     private String messagesBaseUri;
 
     @Autowired
-    private OAuth2RestTemplate messagingClientClientCredsRestTemplate;
+    private OAuth2RestTemplate auth2RestTemplatee;
+
+    @GetMapping(value = "/")
+    public String getHello() {
 
 
+
+        return "hello" ;
+    }
+
     @GetMapping(value = "/authorize", params = "grant_type=client_credentials")
     public String client_credentials_grant(Model model) {
 
         Authentication authentication = SecurityContextHolder.getContext().getAuthentication();
         System.out.println("AUTHENTICATION");
```

Comment:
Jeżeli chcesz coś zalogować korzystaj z dedykowanych bibliotek. Np. `slf4j`

Reply:
Mają one wbudowane poziomy logowania. Możesz logować rzeczy do debugowania z poziomemem debug i zostawić to nawet w kodzie produkcyjnym. Bo on będzie logował np. od poziomu info i te z debug nigdy nie wyskoczą w logach produkcyjnych

Reply:
Poza poziomami logowania w takich bibliotekach można dość dowolnie ustawiać np. gdzie takie logi mają się znaleźć. Mogą iść na {{STDOUT}}, ale też do jakiegoś pliku albo asynchronicznie wysyłane na jakąś kolejkę, gdzie potem będzie to można filtrować i przeglądać

Reply:
Ok, będę pamiętał.